change-viewpoint/src/main/java/com/esri/arcgisruntime/sample/changeviewpoint/MainActivity.java

```diff
@@ -0,0 +1,160 @@
+                        text.append(line);
+                    }
+                } catch (IOException e) {
+                    Log.d(TAG, e.toString());
+                }
+                String JsonString = text.toString();
+
+                // create Geometry from JSON
+                Geometry geometry = Geometry.fromJson(JsonString, spatialReference);
+                GraphicsOverlay overlay = new GraphicsOverlay();
+                // add graphics overlay on map view
+                mMapView.getGraphicsOverlays().add(overlay);
+                SimpleFillSymbol fillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.DIAGONAL_CROSS, Color.GREEN, null);
+                // add graphic of Griffith Park
+                overlay.getGraphics().add(new Graphic(geometry, fillSymbol));
+
+                // set viewpoint of map view to Geometry - Griffith Park
+                mMapView.setViewpointGeometryAsync(geometry);
+
+                mGeometryButton.setBackgroundColor(getResources().getColor(R.color.primary_dark));
+                mAnimateButton.setBackgroundColor(getResources().getColor(R.color.primary));
+                mCenterScaleButton.setBackgroundColor(getResources().getColor(R.color.primary));
+            }
+        });
+
+        mAnimateButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                // create the London location point
+                Point londonPoint = new Point(28677947.756181,22987445.6186465, spatialReference);
+                // create the viewpoint with the London point and scale
+                Viewpoint viewpoint = new Viewpoint(londonPoint, SCALE);
+                // set the map views's viewpoint to London with a ten second duration
+                mMapView.setViewpointWithDurationAsync(viewpoint, 10);
+
+                mGeometryButton.setBackgroundColor(getResources().getColor(R.color.primary));
+                mAnimateButton.setBackgroundColor(getResources().getColor(R.color.primary_dark));
+                mCenterScaleButton.setBackgroundColor(getResources().getColor(R.color.primary));
+            }
+        });
+
+        mCenterScaleButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                // create the Waterloo location point
+                Point waterlooPoint = new Point(28681235.9843606, 22990575.7224154, spatialReference);
+                // set the map views's viewpoint centered on Waterloo and scaled
+                mMapView.setViewpointCenterWithScaleAsync(waterlooPoint, SCALE);
+
+                mGeometryButton.setBackgroundColor(getResources().getColor(R.color.primary));
+                mAnimateButton.setBackgroundColor(getResources().getColor(R.color.primary));
+                mCenterScaleButton.setBackgroundColor(getResources().getColor(R.color.primary_dark));
```

Comment:
Method deprecated, refactor to `ContextCompat.getColor()` for the above 2 lines.